device/linkstate/teprofile.yaml

```diff
@@ -0,0 +1,120 @@
+components:
+  schemas:
+    LinkState.TE:
+      description: >-
+        A container for Traffic Engineering properties on a interface.
+      type: object
+      properties:
+        administrative_group:
+          description: |-
+            The Administrative group sub-TLV (sub-TLV 3). It is a 4-octet 
+            user-defined bit mask used to assign administrative group numbers 
+            to the interface, for use in assigning colors and resource classes. 
+            Each set bit corresponds to a single administrative group for this 
+            interface. The settings translate into Group numbers, which range 
+            from 0 to 31 (integers).
+          type: string
+          format: hex
+          default: '00000000'
+        metric_level:
+          description: |-
+            The user-assigned link metric for Traffic Engineering.
+          type: integer
+          format: int64 
+          minimum: 0
+          maximum: 4261412864
+          default: 0
+        max_bandwith:
+          description: |-
+            The maximum link bandwidth (sub-TLV 9) in bytes/sec allowed for this 
+            link for a direction.
+          type: integer
+          format: int64 
+          minimum: 0
+          maximum: 4294967295
+          default: 125000000
+        max_reservable_bandwidth:
+          description: |-
+            The maximum link bandwidth (sub-TLV 10) in bytes/sec allowed for this 
+            link in this direction.
+          type: integer
+          format: int64 
+          minimum: 0
+          maximum: 4294967295
+          default: 125000000
+        priority_bandwidths_0_7:
```

Comment:
we shouldn't mention `0_7` in name; that's implied by all properties (`pb0`...`pb7` )

Reply:
priority_bandwidths_0_7 mentioned explicitly because there are more bandwidth properties yet to come those have been deleted for first drop.

Reply:
Will the other bandwidth priorities be called `priority_bandwidths_x_y` where x and y are greater than 7 ?
I'm just trying to ensure that later on when adding new priorities, it won't break compatibility.

Reply:
priority_bandwidths_0_7 this is one grp and other b/w properties in different names in the rfc and treated differently.